### DISCUSSION TITLE
Editing Config.yaml to update configuration but no joy ?

### DISCUSSION THREAD

**Discussion by al-jimenez** [Lissy93/dashy#830] [Q&A]
I am new to Dashy, still doing a lot of reading,  but once I got docker image going I can't seem to make changes to the Dashy page by editing the config.yml file.   I can only make changes take via the ui.   Changes to the config.yml will sometimes work after a while but not right away.  A rebuild of the application worked but eliminated the fonts & colors.

Changes to specific colors and fonts wont take at all unless I use the UI.   Am I missing something... drawing a blank...

Any help would be appreciated.

I very much appreciate the depth and breath of this application.  A lot of good work done on it.  Thank you.

**Comment by lissy93** [comment]
#799
